### PR TITLE
Snapshot warning

### PR DESCRIPTION
**Pull request by nacx**
Added reference to the GSoC post in the landing page, and warnings to the snapshot downloads according to the ASF policy for distribution of unreleased material.

### CODE REVIEW

**Comment by jclouds-commentator** [comment]
to review your changes.

**Comment by nacx** [comment]
@zack-shoylev Any updates on the testing account?

**Comment by zack-shoylev** [comment]
It was erased because it went significantly over the credit provided; working on getting a new one now.

File: start/install.md [hunk 1]
```diff
@@ -43,6 +43,10 @@ If you do not have a *pom.xml* file, you can copy and paste the one below. If yo
 
 ### Using the daily builds
 
+<div class="alert alert-danger">
+<strong>Warning!</strong> These are snapshot builds; untested builds provided for your convenience. They have not been tested, and are not official releases of the Apache jclouds project or the Apache Software Foundation.
```
Comment (demobox) [inline start/install.md]: Perhaps shorten a little bit?

"These are untested snapshot builds provided for convenience; they are **not** official releases of the Apache jclouds project, or the Apache Software Foundation."

File: _includes/navbar.html [hunk 2]
```diff
@@ -56,6 +56,7 @@
                     <a href="#" class="dropdown-toggle" data-toggle="dropdown">About Apache<strong class="caret"></strong></a>
                     <ul class="dropdown-menu">
                         <li><a href="http://www.apache.org">The Apache Software Foundation</a></li>
+                        <li><a href="http://www.apache.org/licenses/">License</a></li>
```
Comment (demobox) [inline _includes/navbar.html]: Link to http://www.apache.org/licenses/#2.0, since that's actually the relevant license we are using?

**Comment by demobox** [comment]
> It was erased because it went significantly over the credit provided; working on getting a new one now.

@zack-shoylev: Thanks for the update! Can we put in place some kind of regular purging? We really shouldn't need to store anything in there that's more than a month old, or so - and even that might be a bit long.

@nacx Thanks for taking care of this! Just a couple of minor comments...

**Comment by zack-shoylev** [comment]
@demobox If we want to have any automatic cleanup, we need to make sure to put this in ourselves. I know some of the old jclouds tests didn't do a very good job cleaning up after themselves, especially in case of failures. It's a lot better now, but there might still be some leakage.

**Comment by demobox** [comment]
> If we want to have any automatic cleanup, we need to make sure to put this in ourselves

Yes, sure - sorry if that wasn't clear. I was thinking mainly of the site snapshots, which I thought is all that account was being used for. If there was also test output in there we'd obviously need to look at what's contributing the most to the cost.

Do we know if the ASF has any general Rackspace testing accounts?

/cc @nacx @abayer

File: _includes/navbar.html (hunk 2 above)
Comment (nacx) [inline _includes/navbar.html]: Makes sense. Although this section is more "about the Apache Software Foundation", and not abut jclouds itself. I just added it in accordance to: http://www.apache.org/foundation/marks/pmcs.html#navigation

**Comment by nacx** [comment]
>If we want to have any automatic cleanup, we need to make sure to put this in ourselves

We already have the rackspace cleanup build that runs weekly. Perhaps we should change it to *additionaly* run *after* every run for the rackspace live test builds?

Thanks for taking care of this!

File: start/install.md (hunk 1 above)
Comment (nacx) [inline start/install.md]: Done!

**Comment by jclouds-commentator** [comment]
to review your changes.

File: _includes/navbar.html (hunk 2 above)
Comment (demobox) [inline _includes/navbar.html]: > I just added it in accordance to: http://www.apache.org/foundation/marks/pmcs.html#navigation

Ah, OK - thanks for clarifying. Fine to leave as-is, in that case

**Comment by demobox** [comment]
Thanks for the updates, @nacx - good to go for me!

**Comment by nacx** [comment]
Merged. Thanks!

**Pull request closed without merging**